### PR TITLE
Remove VLV query from EnrollServlet

### PR DESCRIPTION
**Pull request by fmarco76**

### CODE REVIEW

File: base/ca/src/main/java/com/netscape/cms/servlet/cert/EnrollServlet.java [hunk 1]
```diff
@@ -480,23 +480,16 @@ private X509CertInfo[] handleCertAuthDual(X509CertInfo certInfo, AuthToken authT
...
+        if (list.iterator().hasNext()) {
+            X509CertInfo encCertInfo;
+            X509CertInfo[] cInfoArray;
 
-            boolean encCertFound = false;
-
-            while (en.hasMoreElements()) {
-                CertRecord record = en.nextElement();
-                X509CertImpl cert = record.getCertificate();
+            for (CertRecord rec: list) {
```
Comment (edewata) [inline base/ca/src/main/java/com/netscape/cms/servlet/cert/EnrollServlet.java:491]: IIUC the loop will call `list.iterator()` implicitly, so the iterator object will be created twice. Does it matter?

**Review by edewata — APPROVED** [review]
I have one question, but the changes look good.

File: base/ca/src/main/java/com/netscape/cms/servlet/cert/EnrollServlet.java (hunk 1 above)
Comment (fmarco76) [reply base/ca/src/main/java/com/netscape/cms/servlet/cert/EnrollServlet.java]: It is not a problem to generate a new iterator but looking at the code I think we can skip the if altogether.

**Comment by fmarco76** [comment]
@edewata Thanks! I have updated the code removing the `if` statement.

**Pull request merged**